packages/nextcloud/build.yaml

```diff
@@ -12,6 +12,7 @@ targets:
             - public_member_api_docs
             - unreachable_switch_case
             - camel_case_extensions
+            - no_leading_underscores_for_local_identifiers
```

Comment:
Please introduce this in a separate PR as it has nothing to do with the final parameters.